docs/connectors/sources/kafka.md

```diff
@@ -55,6 +55,58 @@ CREATE TABLE INPUT (
 )
 ```
 
+### Starting from a specific offset
```

Comment:
Please also add a table listing all config options toward the start of the document. We should have had it already, but better late than never. For the most part just copying `KafkaInputConfig` comments should do.